content/docs/ai/ai-azure-notebooks.md

```diff
@@ -53,14 +54,14 @@ You can create cells containing Python code that you can run in place by clickin
 
 After you've set up Azure Data Studio and have created a notebook, you can use the following basic example to get started with Neon and `pgvector`.
 
-### Install the psycopg2 driver
+### Install the psycopg driver
```

Comment:
psycopg is newer and easier to install. Psycopg2 fails to install with the python on my mac.

Reply:
Thank you